.devcontainer/advanced-integration/devcontainer.json

```diff
@@ -0,0 +1,39 @@
+// For more details, see https://aka.ms/devcontainer.json.
+{
+	"name": "Advanced Integration",
+	"image": "mcr.microsoft.com/devcontainers/javascript-node:0-20",
+	"workspaceFolder": "/workspaces/${localWorkspaceFolderBasename}/advanced-integration",
```

Comment:
Why set this manually?

Reply:
The repo contains two samples - https://github.com/samruddhikhandale/docs-examples ( [advanced-integration](https://github.com/samruddhikhandale/docs-examples/tree/main/advanced-integration) & [standard-integration](https://github.com/samruddhikhandale/docs-examples/tree/main/standard-integration))

Hence, eventually they would need two dev container files which they could separately link from their guides.

As their requirement only asked to create dev container for one of their samples, I've only added one for now.